plugins/inputs/kafka_consumer/README.md

```diff
@@ -103,6 +103,13 @@ and use the old zookeeper connection method.
   ## '2 * max_processing_time'.
   # max_processing_time = "100ms"
 
+  ## The default number of message bytes to fetch from the broker in each
+  ## request (default 1MB). This should be larger than the majority of
+  ## your messages, or else the consumer will spend a lot of time
+  ## negotiating sizes and not actually consuming. Similar to the JVM's
+  ## `fetch.message.max.bytes`.
+  # fetch_message_bytes = 1048576
```

Comment:
Can you please rename to `consumer_fetch_default` as this is what you ultimately use it as such.